### DISCUSSION TITLE
How to use CLI installed locally?

### DISCUSSION THREAD

**Discussion by aschenkel** [vercel/vercel#4599] [Help] [answered]
On [CLI download page](https://vercel.com/download) you find 
> NOTE: You can also install Vercel CLI inside a project (instead of globally)

Which would be the use steps? 

Installing `yarn add vercel` & then using it from node_modules? how?

Thank you

**Comment by styfle** [comment, marked as answer]
Hi @aschenkel 

If you run `yarn add vercel` in a locally directory, the bin is available in `./node_modules/.bin/vercel`.

For example, `./node_modules/.bin/vc --version` will print the version.

**Reply by aschenkel** [reply]
thats beautiful! thanks a lot for the quick reply
